docs/state-sync.md

```diff
@@ -0,0 +1,168 @@
+# State Sync
+
+## What is State sync?
+
+The Provenance 1.10.0+ release comes with Tendermint Core 0.34 which includes support for state sync. State sync allows a new node to join a network by fetching a snapshot of the application state at a recent height vs. fetching and replaying all historical blocks (which can take days). This can reduce the time needed to sync with the network down to minutes.
+
+<!-- TOC -->
+- [What is State sync?](#what-is-state-sync)
+    - [Starting a new node from State Sync](#starting-a-new-node-from-state-sync)
+        - [Mainnet](#mainnet)
+        - [Testnet](#testnet)
+    - [Power users](#power-users)
+
+
+### Starting a new node from State Sync
+___
+
+#### Mainnet
+
+##### 1. Install the `provenanced` binary
+
+```
+# setup home directory (or directory of your choosing)
+export PIO_HOME=~/.provenanced
+
+# clone and checkout
+git clone https://github.com/provenance-io/provenance.git
+cd provenance
+git checkout tags/v1.10.0 -b v1.10.0
+
+# install binary
+make clean install
+
+# initialize node
+provenanced init choose-a-moniker --chain-id pio-mainnet-1
+curl https://raw.githubusercontent.com/provenance-io/mainnet/main/pio-mainnet-1/genesis.json> genesis.json
```

Comment:
You might want to add a space between genesis.json and > "genesis.json>". It will make it easier to copy.